streams/src/main/java/org/apache/kafka/streams/state/internals/GlobalStateStoreProvider.java

```diff
@@ -41,6 +44,11 @@ public GlobalStateStoreProvider(final Map<String, StateStore> globalStateStores)
         if (!store.isOpen()) {
             throw new InvalidStateStoreException("the state store, " + storeName + ", is not open.");
         }
+        if (store instanceof TimestampedKeyValueStore && queryableStoreType instanceof QueryableStoreTypes.KeyValueStoreType) {
+            return (List<T>) Collections.singletonList(new ReadOnlyKeyValueStoreFacade((TimestampedKeyValueStore<Object, Object>) store));
+        } else if (store instanceof TimestampedWindowStore && queryableStoreType instanceof QueryableStoreTypes.WindowStoreType) {
```

Comment:
super nit: while I don't believe in 100% test coverage, this line isn't covered by the tests

Reply:
Good catch! This slipped. Will add a test (I do believe ;) )